client/extensions/woocommerce/app/store-stats/constants.js

```diff
@@ -53,23 +53,23 @@ export const topProducts = {
 
 export const topCategories = {
 	basePath: '/store/stats/categories',
-	title: translate( 'Categories' ),
+	title: translate( 'Top Categories' ),
 	values: [
 		{ key: 'name', title: translate( 'Title' ) },
 		{ key: 'quantity', title: translate( 'Quantity' ) },
-		{ key: 'total', title: translate( 'Total' ) },
+		{ key: 'total', title: translate( 'Sales' ) },
 	],
 	empty: translate( 'No categories found' ),
 	statType: 'statsTopCategories',
 };
 
 export const topCoupons = {
 	basePath: '/store/stats/coupons',
-	title: translate( 'Coupons' ),
+	title: translate( 'Most Used Coupons' ),
 	values: [
 		{ key: 'name', title: translate( 'Title' ) },
-		{ key: 'quantity', title: translate( 'Quantity' ) },
-		{ key: 'total', title: translate( 'Total' ) },
+		{ key: 'quantity', title: translate( 'Used' ) },
+		{ key: 'total', title: translate( 'Total Discount' ) },
```

Comment:
Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Discount' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wcss-non-day-listviews)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).